src/coreclr/jit/instrsriscv64.h

```diff
@@ -241,6 +241,31 @@ INST(fcvt_d_l,      "fcvt.d.l",       0,   0xd2200053)
 INST(fcvt_d_lu,     "fcvt.d.lu",      0,   0xd2300053)
 INST(fmv_d_x,       "fmv.d.x",        0,   0xf2000053)
 
+// RV32A + RV64A (R-type, R_R_R)
+// For now all of them are seqentially consistent because the Interlocked.* APIs don't expose acquire/release ordering
+INST(lr_w,          "lr.w",           0,   (0b00010 << 27) | (0b11 << 25) | (0b010 << 12) | 0b0101111)
```

Comment:
Could you write hex instruction code like all other instructions and write detail as comment ?
`0b11` is variable part, please write in `emitIns_R_R_R`.

Reply:
I did it to be consistent with the RISC-V ISA spec, they use binary for instructions everywhere and no hex. But I don't have a strong opinion, can be hex + comment.

As for the ordering flags, sure, I can OR them in emitIns_R_R_R.